src/wp-includes/rest-api/fields/class-wp-rest-meta-fields.php

```diff
@@ -478,6 +478,7 @@ protected function get_registered_fields() {
 
 			$default_schema = array(
 				'type'        => $default_args['type'],
+				'label'       => empty( $args['label'] ) ? '' : $args['label'],
```

Comment:
IIRC, it should be `title` in the schema. See: https://github.com/WordPress/wordpress-develop/pull/6495#discussion_r1588903264

Reply:
I see that @Mamaduka is the pioneer in this space and went through the same path for site settings. Brilliant!

Reply:
All thanks to @TimothyBJacobs' guidance!

Reply:
I share this PR in the REST API channel on WP Slack (link requires registration at http://chat.wordpress.org/):

https://wordpress.slack.com/archives/C02RQC26G/p1725559125954679

Reply:
Yep exactly, `title` is what the JSON Schema specifies.

Reply:
I have updated it in this commit: https://github.com/WordPress/wordpress-develop/pull/7298/commits/c307b4fe6f7079c498fa1d20f22c8147676c6449

Thanks for the guidance 🙂